lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -262,7 +262,28 @@ def db_dump(module, target, target_opts="",
         cmd += " {0} ".format(target_opts)
 
     if comp_prog_path:
-        cmd = '{0}|{1} > {2}'.format(cmd, comp_prog_path, pipes.quote(target))
```

Comment:
Can you please add a check if db exists before taking backup ? I see there is a method called `db_exists` which can reused here.

Reply:
Thank you for your advice! I added db_exists() before doing pg_dump and reverted the logic in db_dump().
Please let me know if I made misunderstanding :)